core/loader_shared.c

```diff
@@ -163,6 +165,7 @@ loader_should_call_entry(privmod_t *mod)
 void
 loader_init_prologue(void)
 {
+#ifdef WINDOWS
```

Comment:
Please add comments explaining why the entire innards of this function would be WINDOWS-only.